draft-ietf-uuidrev-rfc4122bis.md

```diff
@@ -1274,7 +1274,7 @@ Fixed-Length Dedicated Counter Seeding:
   counter rollovers.
 
 Fixed-Length Dedicated Counter Length:
-: Care MUST be taken to select a counter bit-length that can properly handle
+: Take care to select a counter bit-length that can properly handle
```

Comment:
```suggestion
: Select a counter bit-length that can properly handle
```

Reply:
Should this be in passive voice like the text above it? Like "Implementations using a fixed-length counter method SHOULD select a counter bit-length that can properly handle the level of timestamp precision in use."

(Sorry, I'm just lurking and cheering you all on! I'm excited for uuidv7!)

Reply:
@greatestape thanks for the suggestion, I think Jim's suggested edit handles it though, I'll proceed with that.